ethbloom/src/lib.rs

```diff
@@ -43,17 +43,14 @@
 //! ```
 //!
 
-#![cfg_attr(not(feature="std"), no_std)]
-
-#[cfg(feature="std")]
-extern crate core;
+#![cfg_attr(not(feature = "std"), no_std)]
+#![allow(clippy::trivially_copy_pass_by_ref)]
```

Comment:
I think it is better to scope this down to the individual functions/methods that do that

Reply:
Done